server/sonar-web/src/main/js/apps/quality-profiles/utils.js

```diff
@@ -42,7 +42,11 @@ export function sortProfiles(profiles: Array<Profile>) {
     children.forEach(child => putProfile(child, depth + 1));
   }
 
-  putProfile();
+  sorted
+    .filter(
+      profile => profile.parentKey == null || sorted.find(p => p.key === profile.parentKey) == null
```

Comment:
A quality profile inheriting from another one is not correctly displayed on the quality profiles page. It should be indented, but it's not. It seem's like it only happens for first level inheritance.